src/settings/MappingProfiles/MappingProfilesForm/MappingProfilesForm.js

```diff
@@ -51,9 +51,11 @@ const MappingProfilesFormComponent = props => {
     isEditMode,
     isFormDirty,
     form,
+    initiallyDisabledRecord,
```

Comment:
Since we changed the naming in the discussion below, let's name this initiallyDisabledRecordTypes both in MappingProfilesFormComponent and in MappingProfilesForm.